ykcapi/src/lib.rs

```diff
@@ -195,15 +185,18 @@ pub extern "C" fn yk_stopgap(
                 assert_eq!(*reg, 6);
                 let addr = unsafe { registers.get(*reg) as *mut u8 };
                 let addr = unsafe { addr.offset(isize::try_from(*off).unwrap()) };
-                let v = match *size {
-                    1 => unsafe { ptr::read::<u8>(addr as *mut u8) as u64 },
-                    2 => unsafe { ptr::read::<u16>(addr as *mut u16) as u64 },
-                    4 => unsafe { ptr::read::<u32>(addr as *mut u32) as u64 },
-                    8 => unsafe { ptr::read::<u64>(addr as *mut u64) as u64 },
-                    _ => unreachable!(),
-                };
+                let aot = &aotmap[i];
+                unsafe {
+                    sginterp.var_init(
+                        aot.bbidx,
+                        aot.instridx,
+                        std::ffi::CStr::from_ptr(aot.fname),
+                        aot.sfidx,
+                        addr as u64,
+                    );
+                }
                 // FIXME: remove once we have a stopgap interpreter.
-                eprintln!("Direct: {} ({} {})", v, reg, off);
+                eprintln!("Direct: {} ({} {}) {}", addr as u64, reg, off, size);
```

Comment:
Since we do have a stopgap interpreter, why do we still have this `eprintln`?

Reply:
I actually think they are still useful. I was planning to put them behind an ENV in a separate PR.

Reply:
I think this sort of uber-low-level-never-useful-to-yk-users debugging should, if not killed entirely (my preference!), put behind a compile-time flag. There's no point us paying a run-time cost for checking for this IMHO.

Reply:
I guess we can kill it. Do you want it in this PR or a separate one?

@vext01 Any thoughts?

Reply:
I don't think there's any determinism guarantees of what we are printing, so testing them is probably not ideal.

I don't feel super-strongly.